style/mono/mono.go

```diff
@@ -0,0 +1,56 @@
+package mono
+
+// The mono theme is a plain default style for monochrome displays.
+
+import (
+	"github.com/aykevl/tinygl"
+	"github.com/aykevl/tinygl/style"
+	"tinygo.org/x/drivers/pixel"
+	font "tinygo.org/x/tinygl-font"
+	"tinygo.org/x/tinygl-font/roboto"
+)
+
+type Mono[T pixel.Color] struct {
+	style.Theme[T]
+}
+
+// NewTheme returns a new basic theme with the given properties.
+//
+// The default style is black text on a white background.
+func NewTheme[T pixel.Color](scale style.Scale, screen *tinygl.Screen[T]) *Mono[T] {
```

Comment:
Maybe add a keyword to this function name? NewTheme sounds like a very generic constructor but this generates a roboto font theme specifically. Maybe `NewRobotoTheme`?

Reply:
That's funny, because I was wondering if it should just be `New()` since you cannot create anything else in the `style` packages. :smile_cat: